calcom/cal.com
# feat: availability in instant meeting

## What does this PR do?



TODO:-


- Fixes  https://github.com/calcom/cal.com/issues/16430 (GitHub issue number)
- Fixes CAL-4215 (Linear issue number - should be visible at the bottom of the GitHub issue description)


https://github.com/user-attachments/assets/8d8d36dd-b827-4c63-82d9-f9fac4531158



## Mandatory Tasks (DO NOT REMOVE)


## How should this be tested?

1) Create a org team event type
2) Enable instant booking in event type settings
3) select any availability from dropdown and click save 
4) Now check the booking page and "Connect Now" modal should be only visible when current time is between the availability selected.
